### PR TITLE
#5365 [Manager] Highlight the unstable workspace

### PR DESCRIPTION
**Pull request by eechih**
### Checklist

- [x] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [x] Add at least one reviewer (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add an assignee (of course, you are the assignee by default)
- [x] Do not set milestone, projects and labels (PR labels are added automatically by Jenkins)
- [x] Link to an issue #5365 
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (If the failed tests are unrelated to this PR, add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/cypress/it/appBar/appBar.test.ts [hunk 1]
```diff
@@ -196,6 +196,13 @@ describe('App Bar', () => {
             `Failed to create workspace ${workspaceName}.`,
           ).should('exist');
         });
+
+        // should be highlight the unstable workspaces
```
Comment (konekoya) [inline ohara-manager/client/cypress/it/appBar/appBar.test.ts:200]: _be_ is not necessary here :)

Comment (eechih) [reply ohara-manager/client/cypress/it/appBar/appBar.test.ts:200]: Got it :)

**Comment by eechih** [comment]
retry manager-it

**Comment by konekoya** [comment]
retry manager-it

**Comment by konekoya** [comment]
@eechih The fail test is a flakey test which will be fixed in #5153 :)

**Comment by eechih** [comment]
> The fail test is a flakey test which will be fixed in #5153 :)

Thanks @konekoya

**Pull request merged**